router/pkg/graphqlschemausage/schemausage_test.go

```diff
@@ -511,6 +565,2548 @@ func TestGetSchemaUsageInfoInterfaces(t *testing.T) {
 	}
 }
 
+// TestInputUsageWithNullVariables verifies that null variable values are tracked with IsNull flag
```

Comment:
What about interfaces and unions? Inline fragments and fragment spreads? Recursive types